pkg/agent/secondarynetwork/podwatch/types.go

```diff
@@ -28,14 +28,11 @@ type IPAMConfig struct {
 }
 
 type SecondaryNetworkConfig struct {
-	CNIVersion string     `json:"cniVersion,omitempty"`
-	Name       string     `json:"name,omitempty"`
-	Type       string     `json:"type,omitempty"`
-	IPAM       IPAMConfig `json:"ipam,omitempty"`
-}
-
-type SecondaryNetworkObject struct {
-	NetworkName   string `json:"name,omitempty"`
-	InterfaceName string `json:"interface,omitempty"`
-	InterfaceType string `json:"type"`
+	CNIVersion string `json:"cniVersion,omitempty"`
+	Name       string `json:"name,omitempty"`
+	// Set type to "antrea"
+	Type string `json:"type,omitempty"`
+	// Set mode to "sriov"
+	Mode string     `json:"mode,omitempty"`
```

Comment:
Do you think `NetworkType` works better? I am not certain. `Mode` works for me too.

Reply:
that works for me. I was inspired by macvlan, but I agree that the macvlan `mode` is a totally different concept :) Let's see if @arunvelayutham likes `networkType` too.

Reply:
NetworkType may be confusing as we are referring to the physical/logical "interface" as Mode (SRIOV or Veth?).

Reply:
I think it is not about interface type (SRIOV or veth), but network type - SRIOV, Overlay, MACVLAN, IPVLAN, etc.